syft/frameworks/torch/hook/hook_args.py

```diff
@@ -254,7 +256,7 @@ def hook_response(attr, response, wrap_type, wrap_args={}, new_self=None):
     return new_response
 
 
-def build_hook_response_function(response, wrap_type, wrap_args):
+def build_hook_for_response_with_function(response, wrap_type, wrap_args):
```

Comment:
maybe `wrap_reponse_in_function`?

Reply:
with the build_ prefix sorry

Reply:
maybe `build_wrap_reponse_with_function` for `build_hook_response_function`?